parameter_files/WebParameters_2dii.yml

```diff
@@ -1,8 +1,8 @@
 default:
     paths:
-        project_location_ext: /Users/jacobkastl/Documents/git/PACTA_analysis/
-        data_location_ext: /Users/jacobkastl/Documents/git/pacta-data/2019Q4/
-        template_location: /Users/jacobkastl/Documents/git/create_interactive_report/
+        project_location_ext: /home/mauro/git/PACTA_analysis/
+        data_location_ext: /home/mauro/git/pacta-data/2019Q4/
+        template_location: /home/mauro/git/create_interactive_report/
```

Comment:
can we use `~/git/PACTA_analysis/` etc. here so that it's portable?

Reply:
The path from home to PACTA_analysis/ and its siblings is generated problematically, in integration-test.Rmd (see `[make_paths_portable()](https://github.com/2DegreesInvesting/PACTA_analysis/blob/current_web_functionality/integration-test.md#configurations)`). This is a horrible hack I came up with to deal with the inflexible way the configuration file is managed. I'm happy to improve that in a separate PR.